.changeset/funny-eggs-sell.md

```diff
@@ -0,0 +1,6 @@
+---
+'@spectrum-web-components/overlay': patch
+'@spectrum-web-components/core': patch
```

Comment:
core doesnt need to be included in this changeset since the only thing added was a copyright